src/promptflow-devkit/promptflow/batch/_single_line_python_executor_proxy.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+from typing import Any, Mapping, Optional
+
+from promptflow.batch._base_executor_proxy import AbstractExecutorProxy
+from promptflow.executor import FlowExecutor
+from promptflow.executor._line_execution_process_pool import LineExecutionProcessPool
+from promptflow.executor._result import LineResult
+from promptflow.storage._run_storage import AbstractRunStorage
+
+
+class SingleLinePythonExecutorProxy(AbstractExecutorProxy):
```

Comment:
add some docstring on 
why adding a new class for SingleLine, instead of adding a PythonExecutorProxy?